tutorials/circuits/3_summary_of_quantum_operations.ipynb

```diff
@@ -124,7 +124,7 @@
     "## Single-Qubit Gates <a name=\"single_gates\"/>\n",
```

Comment:
<p>I think it would be a good idea to include a small note for telling people how to convert the old u1, u2, u3 to the new p and sx gates. There is such information in the release note (you can search on this page: https://qiskit.org/documentation/release_notes.html#notable-changes) Ideally you can do that in a separate PR from this. Would you like to create an issue and a PR for that? Thank you!</p><p> </p><pre class="ql-syntax" spellcheck="false">The QuantumCircuit methods u1, u2 and u3 are now deprecated. Instead the following replacements can be used.

u1(theta) = p(theta) = u(0, 0, theta)
u2(phi, lam) = u(pi/2, phi, lam) = p(pi/2 + phi) sx p(pi/2 lam)
u3(theta, phi, lam) = u(theta, phi, lam) = p(phi + pi) sx p(theta + pi) sx p(lam)

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1329/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Sounds good, and agreed. Will address this on a separate PR based off of #1330